.gitignore

```diff
@@ -25,8 +25,11 @@ package-json.lock
 
 # main binary
 asynqmon
+!cmd/asynqmon
```

Comment:
why do we need this? Don't we want to ignore the binary file?

Reply:
Correct, I think I may have missed it from some intermediate commit, removing it.